dashboard/vendor/assets/javascripts/jquery.overlaps.js

```diff
@@ -0,0 +1,71 @@
+/*! Copyright (c) 2010 Brandon Aaron (http://brandonaaron.net)
+ * Licensed under the MIT License (LICENSE.txt).
+ *
```

Comment:
Makes sense in this case, esp. to clarify the plugin's purpose and contract since they're not clear from the file